Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -38,7 +38,8 @@ constexpr size_t CODE_SIZE = 1024 * 1024 * 32;
 constexpr size_t FARCODE_SIZE = 1024 * 1024 * 64;
 constexpr size_t FARCODE_SIZE_MMU = 1024 * 1024 * 64;
 
-JitArm64::JitArm64() : m_float_emit(this)
+JitArm64::JitArm64(Core::System& system, PowerPC::PowerPCState& ppc_state)
```

Comment:
Is there a reason why we're passing in both? Can we have just a system parameter and then get the ppc_state from that? Same for the classes that inherit from this class.

Reply:
I guess I was still in the mindset of not being able to do that during the construction of System on the Interpreter, but yeah I think that should work here since the Jits are constructed later.